llarp/router_contact.hpp

```diff
@@ -19,7 +19,7 @@
 
 namespace lokimq
 {
-  struct bt_list_consumer;
+  class bt_list_consumer;
```

Comment:
for some reason I conflated this with the list producer I was working on when you mentioned it, so I was confused.  ty for fixing.

Reply:
yw